### PR TITLE
Fixes `toggle-left` & `toggle-right`

### PR DESCRIPTION
**Pull request by karsa-mistmere**
image

### CODE REVIEW

File: icons/toggle-left.svg [hunk 1]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="1" y="5" width="22" height="14" rx="7" ry="7" />
-  <circle cx="8" cy="12" r="3" />
+  <rect x="2" y="6" width="20" height="12" rx="6"/>
+  <circle cx="8" cy="12" r="2"/>
```
Comment (karsa-mistmere) [inline icons/toggle-left.svg:13]:
```suggestion
  <rect x="2" y="6" width="20" height="12" rx="6" />
  <circle cx="8" cy="12" r="2" />
```

File: icons/toggle-right.svg [hunk 2]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="1" y="5" width="22" height="14" rx="7" ry="7" />
-  <circle cx="16" cy="12" r="3" />
+  <rect x="2" y="6" width="20" height="12" rx="6"/>
+  <circle cx="16" cy="12" r="2"/>
```
Comment (karsa-mistmere) [inline icons/toggle-right.svg:13]:
```suggestion
  <rect x="2" y="6" width="20" height="12" rx="6" />
  <circle cx="16" cy="12" r="2" />
```

File: icons/toggle-left.svg [hunk 3]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="1" y="5" width="22" height="14" rx="7" ry="7" />
-  <circle cx="8" cy="12" r="3" />
+  <rect x="2" y="6" width="20" height="12" rx="6" />
```
Comment (karsa-mistmere) [inline icons/toggle-left.svg:12]:
```suggestion
  <rect x="2" y="6" width="20" height="12" rx="6" ry="6" />
```

File: icons/toggle-right.svg [hunk 4]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="1" y="5" width="22" height="14" rx="7" ry="7" />
-  <circle cx="16" cy="12" r="3" />
+  <rect x="2" y="6" width="20" height="12" rx="6" />
```
Comment (karsa-mistmere) [inline icons/toggle-right.svg:12]:
```suggestion
  <rect x="2" y="6" width="20" height="12" rx="6" ry="6" />
```

**Comment by ericfennis** [comment]
Maybe we can also make an icon with to switches stacked on each other. 🤔 What do you think?

**Comment by mittalyashu** [comment]
Due to the smaller size of the inner circle it doesn't give a proper visual representation of the toggle.

@karsa-mistmere Can you please try to increase the height of the toggle, that will give more space for the inner circle to grow in both height/width.

**Comment by karsa-mistmere** [comment]
Here's a comparison:
(1st column original, 2nd column PR version, 3rd column @mittalyashu's proposal, 4th column an alternate proposal)
toggles

**Comment by mittalyashu** [comment]
1st or 3rd column looks good.

**Comment by lscheibel** [comment]
I also like the 4th option

**Comment by ericfennis** [comment]
I like the 4th one as well. Maybe change it?

**Pull request merged**